browser/lib/consts.js

```diff
@@ -12,6 +12,16 @@ const themes = fs.readdirSync(themePath)
   })
 themes.splice(themes.indexOf('solarized'), 1, 'solarized dark', 'solarized light')
 
+const snippetFile = process.env.NODE_ENV === 'production'
+  ? path.join(app.getPath('appData'), 'Boostnote', 'snippets.json')
+  : path.join(getAppData(), 'Boostnote', 'snippets.json')
+
+function getAppData () {
+  return process.env.APPDATA || (process.platform === 'darwin'
+  ? process.env.HOME + 'Library/Preferences'
+  : require('os').homedir() + '/.config')
```

Comment:
path.sep woulde be way better than a hardcoded '/' since that isn't platformindependent!

Reply:
First time I heard of `path.sep` new stuff to learn! thanks!

Reply:
Using `path.sep` is not the best. Please use `path.join()` to concatenating path.